frontend/internal-packages/storybook/package.json

```
@@ -0,0 +1,26 @@
+{
+  "name": "@liam-hq/storybook",
+  "private": true,
+  "version": "0.1.0",
+  "dependencies": {
+    "@liam-hq/ui": "workspace:*",
+    "react": "18.3.1",
+    "react-dom": "18"
+  },
+  "devDependencies": {
+    "@storybook/addon-essentials": "8.6.12",
+    "@storybook/addon-interactions": "8.6.12",
+    "@storybook/addon-links": "8.6.12",
+    "@storybook/nextjs": "8.6.12",
+    "@storybook/react": "8.6.12",
+    "@types/react": "18",
+    "@types/react-dom": "18",
+    "storybook": "8.6.12",
+    "typescript": "5.8.3"
+  },
+  "scripts": {
+    "build": "storybook build",
+    "dev": "storybook dev -p 6006",
+    "storybook": "storybook dev -p 6006"
```

Can we unify to dev?

```suggestion
    "dev": "storybook dev -p 6006"
```